lib/explorer.js

```diff
@@ -54,13 +54,13 @@ function executeCleanUp () {
   })
 }
 
-function Explorer (port) {
+function Explorer (port, options) {
```

Comment:
i'd suggest to rename `options` to `pingOptions`. (two times `port` for example might be a bit confusing :) )